onboarding/main.go

```diff
@@ -0,0 +1,138 @@
+package main
+
+import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"os"
+
+	"github.com/red-hat-storage/ocs-operator/v4/controllers/util"
+	"golang.org/x/net/context"
+	corev1 "k8s.io/api/core/v1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog/v2"
+	runtime "sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+const (
+	onboardingTicketPublicKeySecretName  = "onboarding-ticket-key" //Name of existing public key which is used ocs-operator
+	onboardingTicketPrivateKeySecretName = "onboarding-ticket-private-key"
+	serviceAccountName                   = "onboarding-secret-generator"
+)
+
+func main() {
+
+	clientset, err := newClient()
+	if err != nil {
+		klog.Error(err, "failed to create controller-runtime client")
+		return
+	}
+
+	operatorNamespace, err := util.GetOperatorNamespace()
```

Comment:
I don't see an env set for this function to get the value.

Reply:
Yes correct now i have set the variable in pod spec